### ISSUE TITLE
[HL2] In Episode One (ep1_c17_02b), Alyx can get permanently stuck in a locker.

### ISSUE THREAD

**Issue by Evisneffo** [ValveSoftware/Source-1-Games#6602]
![20241117171639_1](https://github.com/user-attachments/assets/44367e07-8f89-4b9a-8a68-7627d3113604)
The locker is located in the mined room before exiting to the surface (map ep1_c17_02b).

**Comment by misyltoad** [comment]
I am curious to see if this is impacted at all if you try the "beta" branch.

**Comment by Evisneffo** [comment]
> ep1_c17_02b

I was able to reproduce this bug in the current updated version and found that Alyx gets stuck in the locker if its right door is open.

After this, I tested the old beta version (Steam Legacy - Pre-20th Anniversary Build) and could not reproduce the bug there. Alyx does not get stuck in that version.

**Comment by Evisneffo** [comment]
> I am curious to see if this is impacted at all if you try the "beta" branch.

@misyltoad, I was able to reproduce this bug again in the current updated version and found that Alyx gets stuck in the locker if its right door is open. Once she is stuck, closing and/or reopening the door does not resolve the issue.

After this, I tested the old beta version (Steam Legacy - Pre-20th Anniversary Build) and could not reproduce the bug there. Alyx does not get stuck in that version.

**Comment by Evisneffo** [comment]
> I am curious to see if this is impacted at all if you try the "beta" branch.

@misyltoad, I checked the branch named "beta - Beta updates," and this bug is also present there. 
![image](https://github.com/user-attachments/assets/66953b53-9cbc-49dd-817c-24fbc609dded)

**Comment by Evisneffo** [comment]
> I am curious to see if this is impacted at all if you try the "beta" branch.

@misyltoad, The bug is also present in the "previous - The previous public build" branch.
![image](https://github.com/user-attachments/assets/87d9f110-0c9d-4a61-bfa7-b35cfaf99c0f)

**Comment by Evisneffo** [comment]
> After this, I tested the old beta version (Steam Legacy - Pre-20th Anniversary Build) and could not reproduce the bug there. Alyx does not get stuck in that version.

@misyltoad, I was mistaken in thinking this issue didn't exist in the older version of the game. I just launched Episode One separately and was able to reproduce the bug there as well.
![image](https://github.com/user-attachments/assets/bcaf7cb6-acc8-48ef-ae89-16759b3a401d)

Then, I switched to the "Steam Legacy - Pre-20th Anniversary Build" branch and launched Episode One separately again, and the bug is present there as well.
![image](https://github.com/user-attachments/assets/d6a5f3da-9ca2-4a1b-b255-6b0b1a2a23ff)

Simply put, the bug is present in all versions of the game currently available.
